packages/vscode-extension/tests/specs/api/autoCompletion.spec.ts

```diff
@@ -246,45 +247,6 @@ suite('Auto completion spec', () => {
     });
   });
 
-  test('Parameters are available in completions even when disconnected from neo4j', async () => {
-    await vscode.commands.executeCommand(
-      CONSTANTS.COMMANDS.INTERNAL.EVAL_PARAMETER,
-      'a',
-      '"charmander"',
-    );
-
-    await vscode.commands.executeCommand(
-      CONSTANTS.COMMANDS.INTERNAL.EVAL_PARAMETER,
-      'b',
-      '"pikachu"',
-    );
-    await vscode.commands.executeCommand(
-      CONSTANTS.COMMANDS.INTERNAL.FORCE_DISCONNECT,
-    );
-    const textDocument = await newUntitledFileWithContent(`RETURN `);
-    const position = new vscode.Position(0, 7);
-    const expecations: vscode.CompletionItem[] = [
-      {
-        label: '$a',
-        kind: vscode.CompletionItemKind.Variable,
-      },
-      {
-        label: '$b',
-        kind: vscode.CompletionItemKind.Variable,
-      },
-    ];
-    await testCompletionContains({
-      textFile: textDocument.uri,
-      position: position,
-      expected: expecations,
-    });
-
-    await vscode.commands.executeCommand(
-      CONSTANTS.COMMANDS.INTERNAL.FORCE_CONNECT,
-      0,
-    );
```

Comment:
I think there's a race condition between this and the evaluate parameter of the next test. Just because when we connect to a new connection, I think we might not immediately set the `schemaPoller.connection` inside it?

Reply:
So I've swapped the order of the tests, but not sure if there's something else more elegant we could do